### PR TITLE
Second try at i18n, without breaking index.html

### PR DESCRIPTION
**Pull request by raindrift**
This is another try at #164, but also updates index.html to load the i18n library.

### CODE REVIEW

File: public/give.html [hunk 1]
```diff
@@ -140,36 +145,46 @@
...
+    <!-- Make sure to translate these meta tags when creating a new language version -->
+
+    <!-- English -->
+    <meta lang="en" property="og:title" content="#findthemasks"/>
+    <meta lang="en" property="og:url" content="https://findthemasks.com/"/>
+    <meta lang="en" property="og:description" content="America’s frontline healthcare workers are treating COVID-19 patients without adequate protective gear, risking their lives! We need to find the masks. All of these masks can save lives now if you get them into the hands of healthcare workers."/>
+
+    <!-- French -->
+    <meta lang="fr" property="og:title" content="#findthemasks"/>
+    <meta lang="fr" property="og:url" content="https://findthemasks.com/"/>
```
Comment (drwestco) [inline public/give.html]: If title/url don't change, do you need separate tags per language?

**Comment by nburt** [comment]
@raindrift fyi I added a couple strings in stats.html and stats.js which was just merged

File: public/give.html [hunk 2]
```diff
@@ -140,36 +145,46 @@
...
+
     <meta property="og:image" content="https://findthemasks.com/images/n95.jpg"/>
 </head>
 
 <body>
 <main class="content">
     <div class="container" style="padding-bottom: 40px;">
       <div class="map-container" style="display: none;">
-        <h2 style="padding: 30px 0px 20px 0px;">Map of donation sites</h2>
+        <h2 style="padding: 30px 0px 20px 0px;" class='i18n' data-i18n="ftm-map-of-sites">Map of sites™</h2>
```
Comment (graemer) [inline public/give.html]: @raindrift - what's going on here with the TM? and change to donation?

File: public/give.html [hunk 3]
```diff
@@ -140,36 +145,46 @@
...
         <div id="map"></div>
       </div>
 
       <div class="locations-container" style="display: none;">
-        <h2 style="padding: 30px 0px 5px 0px; margin: 0px;">List of donation sites</h2>
-        <p style="padding-bottom: 10px">To add a donation site, update information, or request we take down information, please
-          <a href="https://docs.google.com/forms/d/e/1FAIpQLSfgCpK5coPVFC6rJrE7ZhimiZuDoEaL6fo6gYqxsN_FIpJZhg/viewform">fill in this form</a>.</p>
+        <h2 style="padding: 30px 0px 5px 0px; margin: 0px;" class='i18n' data-i18n="ftm-list-of-sites">List of donation sites</h2>
+        <p style="padding-bottom: 10px"> <span class='i18n' data-i18n="ftm-request-update">To add a donation site, update information, or request we take down information, please</span>
+          <a href="https://docs.google.com/forms/d/e/1FAIpQLSfgCpK5coPVFC6rJrE7ZhimiZuDoEaL6fo6gYqxsN_FIpJZhg/viewform"><span class='i18n' data-i18n="ftm-fill-in-this-form">fill in this form</span></a>.</p>
```
Comment (graemer) [inline public/give.html]: You sure there's going to be a space in between "please" and "fill out this form"?

File: public/give.html (hunk 2 above)
Comment (graemer) [reply public/give.html]:
```suggestion
        <h2 style="padding: 30px 0px 20px 0px;" class='i18n' data-i18n="ftm-map-of-sites">Map of donation sites</h2>
```

**Pull request merged**